### PR TITLE
Move to helm chart (deploy to new cluster)

### PR DESCRIPTION
**Pull request by torbencarstens**
**Don't simply merge this, I need to create the PVC and transfer the state manually before this can be deployed**

I've overwritten the `KUBECONFIG_RAW` org secret with a repo secret, the `KUBECONFIG_RAW` secret in this repo is configured to deploy to the new cluster.

output of dry_run (`helm upgrade -n telegram-bots hhhdiffbot . --install --dry-run`):

```yml
---
# Source: hhh-diff-bot/templates/secret.yml
apiVersion: v1
kind: Secret
metadata:
  name: hhhdiffbot-telegram-token
  namespace: telegram-bots
data:
  DEFAULT_PASSWORD: "__TELEGRAM_TOKEN__"
---
# Source: hhh-diff-bot/templates/pvc.yml
apiVersion: v1
kind: PersistentVolumeClaim
metadata:
  name: hhhdiffbot-state
  namespace: telegram-bots
spec:
  accessModes:
    - ReadWriteOnce
  resources:
    requests:
      storage: 1Gi
---
# Source: hhh-diff-bot/templates/deployment.yaml
apiVersion: apps/v1
kind: Deployment
metadata:
  name: hhhdiffbot
  namespace: telegram-bots
spec:
  revisionHistoryLimit: 1
  replicas: 1
  selector:
    matchLabels:
      app: hhhdiffbot
  template:
    metadata:
      labels:
        app: hhhdiffbot
    spec:
      containers:
      - name: hhhdiffbot
        image: ghcr.io/preparingforexams/hhh-diff-bot:__TAG__
        imagePullPolicy: Always
        envFrom:
          - secretRef:
              name: hhhdiffbot-telegram-token
        volumeMounts:
          - mountPath: /data
            name: hhhdiffbot-state
        resources:
          requests:
            cpu: 50m
            memory: 50Mi
      volumes:
      - name: hhhdiffbot-state
        persistentVolumeClaim:
          claimName: hhhdiffbot-state
```

### CODE REVIEW

File: values.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+namespace: telegram-bots
+
+deployment:
+  name: hhhdiffbot
+  image: ghcr.io/preparingforexams/hhh-diff-bot:__TAG__
+  imagePullPolicy: Always
```
Comment (BjoernPetersen) [inline values.yaml:6]: Why not `IfNotPresent`?

**Review by BjoernPetersen — APPROVED** [review]
I find your heavy use of Helm templates more confusing than helpful, but generally I approve.

File: values.yaml (hunk 1 above)
Comment (torbencarstens) [reply values.yaml]: No special reason, just update it

**Comment by torbencarstens** [comment]
> I find your heavy usage of Helm templates more confusing than helpful, but generally I approve.

Makes it easier to be reusable across projects IMO

**Pull request merged**